otter/test/convergence/test_service.py

```diff
@@ -330,13 +359,13 @@ def test_unexpected_errors(self):
         dirty flag.
         """
         expected_error = RuntimeError('oh no!')
-        sequence = SequenceDispatcher([
+        sequence = [
```

Comment:
Non-blocking - just wondering if it's worth converting this test (and maybe the failure test) to not use the fallback dispatcher, to ensure that if there is an error during convergence that `recently_converged` is not updated, and also that if the convergence finishes but fails, `recently_converged` *is* updated?